### PR TITLE
Metadata

### PR DESCRIPTION
**Pull request by snwhite**
Updates to the eng_bulk_load CSV for CGSN controller components as a part of the metadata review.  A spreadsheet of the changes was previously emailed out for review (eng_bulk_load-AssetRecord_NEW_2019-05-17.xlsx)

### CODE REVIEW

File: bulk/eng_bulk_load-AssetRecord.csv [hunk 1]
```diff
@@ -4,116 +4,122 @@ CE02SHBP-MJ01C-05-IP1,,Sensor,0,JBox Port J05 (LJ01D),JBox Port J05 (LJ01D),,,,,
...
+CGCON-BCPM01-50008,,Sensor,0,Surface Buoy Communications and Power Manager,Communications and Power Manager,WHOI,CPM,2015-0008,,,,CE09OSSM-00005-SBC11,Oregon State University
+CGCON-BCPM01-50009,,Sensor,0,Surface Buoy Communications and Power Manager,Communications and Power Manager,WHOI,CPM,2015-0009,,,,,Woods Hole Oceanographic Institution
+CGCON-BCPM01-60002,OL000593,Sensor,0,Surface Buoy Communications and Power Manager,Communications and Power Manager,WHOI,CPM,2016-0002,,,,,Woods Hole Oceanographic Institution
+CGCON-BCPM01-60701,,Sensor,0,Surface Buoy Communications and Power Manager,Communications and Power Manager,WHOI,CPM,60701,,,,DELETE?,
+CGCON-BCPM01-70001,,Sensor,0,Surface Buoy Communications and Power Manager,Communications and Power Manager,WHOI,CPM,2017-0001,,,,CP04OSSM-00006-CPM1,Woods Hole Oceanographic Institution
+CGCON-BCPM01-70009,CGCON-BCPM01-17009,Sensor,0,Surface Buoy Communications and Power Manager,Communications and Power Manager,WHOI,CPM,2017-0009,,,,,Woods Hole Oceanographic Institution
+CGCON-BCPM01-70009,,Sensor,0,Surface Buoy Communications and Power Manager,Communications and Power Manager,WHOI,CPM,UNKNOWN,,,,,Woods Hole Oceanographic Institution
```
Comment (rtravis-whoi) [inline bulk/eng_bulk_load-AssetRecord.csv]: I believe the ASSET_UID for this one is supposed to be CGCON-BCPM01-99999 (according to eng_bulk_load-AssetRecord_NEW_2019-05-17 spreadsheet.

File: bulk/eng_bulk_load-AssetRecord.csv [hunk 2]
```diff
@@ -4,116 +4,122 @@ CE02SHBP-MJ01C-05-IP1,,Sensor,0,JBox Port J05 (LJ01D),JBox Port J05 (LJ01D),,,,,
...
+CGCON-BDCL11-50007,OL000324,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2015-0007,,,,CE02SHSM-00003-DCL11,Oregon State University
+CGCON-BDCL11-50008,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2015-0008,,,,CE09OSSM-00005-SBD11,Oregon State University
+CGCON-BDCL11-50009,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2015-0009,,,,,Woods Hole Oceanographic Institution
+CGCON-BDCL11-60002,OL000594,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2016-0002,,,,,Woods Hole Oceanographic Institution
+CGCON-BDCL11-60701,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,60701,,,,DELET?,
+CGCON-BDCL11-70001,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2017-0001,,,,CP04OSSM-00006-DCL11,Woods Hole Oceanographic Institution
+CGCON-BDCL11-70009,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2017-0009,,,,,Woods Hole Oceanographic Institution
```
Comment (rtravis-whoi) [inline bulk/eng_bulk_load-AssetRecord.csv]: eng_bulk_load-AssetRecord_NEW_2019-05-17 spreadsheet shows manuf sn as 2017-009; not sure if spreadsheet or this intry in csv is incorrect.

File: bulk/eng_bulk_load-AssetRecord.csv [hunk 3]
```diff
@@ -4,116 +4,122 @@ CE02SHBP-MJ01C-05-IP1,,Sensor,0,JBox Port J05 (LJ01D),JBox Port J05 (LJ01D),,,,,
...
+CGCON-BDCL12-00001,OL000487,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,001,,,,CP01CNSM-00001-DCL12,Woods Hole Oceanographic Institution
+CGCON-BDCL12-00002,OL000438,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,3703-00235-0002,,,,CE07SHSM-00003-DCL12,Oregon State University
+CGCON-BDCL12-00004,OL000501,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,004,,,,CP01CNSM-00002-DCL12,Woods Hole Oceanographic Institution
+CGCON-BDCL12-00006,OL000382 OL000534 OL000558,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,006,,,,CP04OSSM-00003-DCL12,Woods Hole Oceanographic Institution
+CGCON-BDCL12-00007,OL000546,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,007,,,,,Woods Hole Oceanographic Institution
+CGCON-BDCL12-00008,OL000393 OL000570 R00019,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,008,,,,CE04OSSM-00002-SBD12,Oregon State University; Woods Hole Oceanographic Institution
```
Comment (rtravis-whoi) [inline bulk/eng_bulk_load-AssetRecord.csv]: eng_bulk_load-AssetRecord_NEW_2019-05-17 spreadsheet shows 2nd entry for mooring info: CP03ISSM-00001-DCL12

File: bulk/eng_bulk_load-AssetRecord.csv [hunk 4]
```diff
@@ -4,116 +4,122 @@ CE02SHBP-MJ01C-05-IP1,,Sensor,0,JBox Port J05 (LJ01D),JBox Port J05 (LJ01D),,,,,
...
+CGCON-BDCL12-50008,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2015-0008,,,,CE09OSSM-00005-SBD12,Oregon State University
+CGCON-BDCL12-50009,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2015-0009,,,,,Woods Hole Oceanographic Institution
+CGCON-BDCL12-60002,OL000595,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2016-0002,,,,,Woods Hole Oceanographic Institution
+CGCON-BDCL12-60701,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,60701,,,,DELETE?,
+CGCON-BDCL12-70001,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2017-0001,,,,CP04OSSM-00006-DCL12,Woods Hole Oceanographic Institution
+CGCON-BDCL12-70009,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2017-009,,,,,Woods Hole Oceanographic Institution
+CGCON-BDCL12-70009,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,UNKNOWN,,,,,Woods Hole Oceanographic Institution
```
Comment (rtravis-whoi) [inline bulk/eng_bulk_load-AssetRecord.csv]: I believe the ASSET_UID for this one is supposed to be CGCON-BDCL12-99999 (according to eng_bulk_load-AssetRecord_NEW_2019-05-17 spreadsheet.

File: bulk/eng_bulk_load-AssetRecord.csv [hunk 5]
```diff
@@ -122,465 +128,481 @@ CGCON-CWFPCP-10302,A00847.1,Sensor,0,Coastal Wire-Following Profiler Controller,
...
-CGCON-GLDRCP-00380,OL000355,Sensor,1,Mobile Asset Controller for Coastal Glider,GL380 Controller,Teledyne Webb,G2 Slocum Gliders,380,,,,,
-CGCON-GLDRCP-00387,OL000356,Sensor,1,Mobile Asset Controller for Coastal Glider,GL387 Controller,Teledyne Webb,G2 Slocum Gliders,387,,,,,
-CGCON-GLDRCP-00388,OL000357,Sensor,1,Mobile Asset Controller for Coastal Glider,GL388 Controller,Teledyne Webb,G2 Slocum Gliders,388,,,,,
-CGCON-GLDRCP-00389,OL000358,Sensor,1,Mobile Asset Controller for Coastal Glider,GL389 Controller,Teledyne Webb,G2 Slocum Gliders,389,,,,,
-CGCON-GLDRCP-00564,,Sensor,1,Mobile Asset Controller for Coastal Glider,GL564 Controller,Teledyne Webb,G2 Slocum Gliders,564,,,,,
-CGCON-GLDRGP-00470,,Sensor,1,Mobile Asset Controller for Profiling Glider,,Teledyne Webb,G2 Slocum Gliders,470,,,,,Woods Hole Oceanographic Institution
+CGCON-CWFPCP-99105,,Sensor,0,Coastal Wire-Following Profiler Controller,WFP COASTAL Electronics #1,McLane,MMP-7,ML12991-05,,20130510,0,Early on, this WFP had controller brains swapped with 12991-03 (belonging to EA),Woods Hole Oceanographic Institution
```
Comment (rtravis-whoi) [inline bulk/eng_bulk_load-AssetRecord.csv]: Does it matter that the comment has a comma?  (Early on, this WFP...)

File: bulk/eng_bulk_load-AssetRecord.csv [hunk 6]
```diff
@@ -122,465 +128,481 @@ CGCON-CWFPCP-10302,A00847.1,Sensor,0,Coastal Wire-Following Profiler Controller,
...
+CGCON-ECPM01-00002,OL000201,Sensor,0,Surface Buoy Communications and Power Manager,Communications and Power Manager,WHOI,CPM,2,,,,"CE06ISSM-00001-CPM1 CE01ISSM-0000(1,2,4)-CPM1",Oregon State University
+CGCON-ECPM01-00015,OL000208,Sensor,0,Surface Buoy Communications and Power Manager,Communications and Power Manager,WHOI,CPM,15,,,,CE06ISSM-00004 CE06ISSM-00002-CPM1,Oregon State University
+CGCON-ECPM01-00016,OL000295,Sensor,0,Surface Buoy Communications and Power Manager,Communications and Power Manager,WHOI,CPM,3605-20018-0016,,,,"CE01ISSM-0000(3,5)-CPM1",Oregon State University
+CGCON-ECPM01-00020,,Sensor,0,Surface Buoy Communications and Power Manager,Communications and Power Manager,WHOI,CPM,20,,,,CE01ISSM-00006-SBC11,Oregon State University
+CGCON-ECPM01-01701,,Sensor,0,Surface Buoy Communications and Power Manager,Communications and Power Manager,WHOI,CPM,1701,,,,,
```
Comment (rtravis-whoi) [inline bulk/eng_bulk_load-AssetRecord.csv]: According to eng_bulk_load-AssetRecord_NEW_2019-05-17 spreadsheet this entry should have Oregon State University added for MIO

File: bulk/eng_bulk_load-AssetRecord.csv [hunk 7]
```diff
@@ -122,465 +128,481 @@ CGCON-CWFPCP-10302,A00847.1,Sensor,0,Coastal Wire-Following Profiler Controller,
...
+CGCON-ECPM01-50002,OL000216,Sensor,0,Surface Buoy Communications and Power Manager,Communications and Power Manager,WHOI,CPM,2,,,,CE06ISSM-00001-CPM1  added 50000 to sn UID  to distinguish the duplicated unit beteween with CE01ISSM-00002,Oregon State University
+CGCON-EDCL17-00002,OL000203,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2,,,," CE06ISSM-00003-DCL17 CE01ISSM-0000(1,2,4)-DCL17 ",Oregon State University
+CGCON-EDCL17-00015,OL000210,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,15,,,,"CE06ISSM-0000(2,4)-DCL17",Oregon State University
+CGCON-EDCL17-00016,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,16,,,,CE06ISSM-00006-DCL17,Oregon State University
+CGCON-EDCL17-00020,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,20,,,,CE01ISSM-00006-DCL17,Oregon State University
+CGCON-EDCL17-01701,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,1701,,,,,
```
Comment (rtravis-whoi) [inline bulk/eng_bulk_load-AssetRecord.csv]: According to eng_bulk_load-AssetRecord_NEW_2019-05-17 spreadsheet this entry should have Oregon State University added for MIO

File: bulk/eng_bulk_load-AssetRecord.csv [hunk 8]
```diff
@@ -122,465 +128,481 @@ CGCON-CWFPCP-10302,A00847.1,Sensor,0,Coastal Wire-Following Profiler Controller,
...
-CGCON-MDCL35-00032,,Sensor,0,Multi-Function Node Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,32,,,,CE01ISSM-00007-DCL35,
+CGCON-MCPM03-00020,,Sensor,0,Multi-Function Node Communications and Power Manager,Communications and Power Manager,WHOI,CPM,20,,,,,
+CGCON-MCPM03-50001,OL000280,Sensor,0,Multi-Function Node Communications and Power Manager,Communications and Power Manager,WHOI,CPM,2015-0001,,,,CP04OSSM-00004-CPM3 TO BE DELETED,
+CGCON-MCPM03-50004,OL000262,Sensor,0,Multi-Function Node Communications and Power Manager,Communications and Power Manager,WHOI,CPM,2015-0004,,,,CP03ISSM-00004-CPM3 TO BE DELETED,
+CGCON-MCPM03-50005,OL000230,Sensor,0,Multi-Function Node Communications and Power Manager,Communications and Power Manager,WHOI,CPM,2015-0005,,,,CP01CNSM-00005-CPM31 TO BE DELETED,
+CGCON-MDCL35-00001,OL000444,Sensor,0,Multi-Function Node Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,3703-00292-0001,,,,CE07SHSM-00003-DCL35,Oregon State University
+CGCON-MDCL35-00002,OL000184,Sensor,0,Multi-Function Node Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2,,,,CE01ISSM-00001-DCL35,Oregon State University
```
Comment (rtravis-whoi) [inline bulk/eng_bulk_load-AssetRecord.csv]: According to eng_bulk_load-AssetRecord_NEW_2019-05-17 spreadsheet Manuf Serial No. should be changed to 0002

File: bulk/eng_bulk_load-AssetRecord.csv [hunk 9]
```diff
@@ -122,465 +128,481 @@ CGCON-CWFPCP-10302,A00847.1,Sensor,0,Coastal Wire-Following Profiler Controller,
...
+CGCON-MDCL35-00009,OL000537,Sensor,0,Multi-Function Node Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,0009,,,,CP04OSSM-00001-DCL35,Woods Hole Oceanographic Institution
+CGCON-MDCL35-00010,OL000404,Sensor,0,Multi-Function Node Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,3703-00292-0010,,,,CE06ISSM-00004,Oregon State University
+CGCON-MDCL35-00011,OL000196,Sensor,0,Multi-Function Node Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,11,,,,CE01ISSM-00003-DCL35,Oregon State University
+CGCON-MDCL35-00012,OL000285,Sensor,0,Multi-Function Node Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,0012,,,,"CP04OSSM-MFD35-00-DCLENG000,4",Woods Hole Oceanographic Institution
+CGCON-MDCL35-00013,OL000267,Sensor,0,Multi-Function Node Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,0013,,,,"CP03ISSM-MFD35-00-DCLENG000,4",Woods Hole Oceanographic Institution
+CGCON-MDCL35-00014,,Sensor,0,Multi-Function Node Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,14,,,,CE01ISSM-00006-DCL35,Oregon State University
```
Comment (rtravis-whoi) [inline bulk/eng_bulk_load-AssetRecord.csv]: According to eng_bulk_load-AssetRecord_NEW_2019-05-17 spreadsheet the Manuf Serial No. should be updated to 0014

File: bulk/eng_bulk_load-AssetRecord.csv [hunk 10]
```diff
@@ -122,465 +128,481 @@ CGCON-CWFPCP-10302,A00847.1,Sensor,0,Coastal Wire-Following Profiler Controller,
...
-CGCON-NDCL26-00004,OL000488,Sensor,0,Near Surface Instrument Frame Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,4,,,,CP01CNSM-00001-DCL26,
-CGCON-NDCL26-00006,,Sensor,0,Near Surface Instrument Frame Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,6,,,,,
-CGCON-NDCL26-00007,OL000502,Sensor,0,Near Surface Instrument Frame Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,7,,,,CP01CNSM-00002-DCL26,
-CGCON-NDCL26-00009,,Sensor,0,Near Surface Instrument Frame Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,9,,,,CE07SHSM-00004-DCL26,
-CGCON-NDCL26-00010,,Sensor,0,Near Surface Instrument Frame Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,3703-00272-0010,,,,CE02SHSM-00005-DCL26,
-CGCON-NDCL26-00011,OL000571,Sensor,0,Near Surface Instrument Frame Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,11,,,,CP03ISSM-00001-DCL26,
-CGCON-NDCL26-00013,R00032,Sensor,0,Near Surface Instrument Frame Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,13,,,,CP03ISSM-00003-DCL26,
-CGCON-NDCL26-00014,OL000441,Sensor,0,Near Surface Instrument Frame Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,3703-00
```
Comment (rtravis-whoi) [inline bulk/eng_bulk_load-AssetRecord.csv]: eng_bulk_load-AssetRecord_NEW_2019-05-17 spreadsheet shows manuf sn as 0001; not sure if spreadsheet or this entry in csv is incorrect.

Comment (rtravis-whoi) [inline bulk/eng_bulk_load-AssetRecord.csv]: eng_bulk_load-AssetRecord_NEW_2019-05-17 spreadsheet shows manuf sn as 00011; I believe that 0011 is correct in this csv, but should verify whether spreadsheet or this is correct.

Comment (rtravis-whoi) [inline bulk/eng_bulk_load-AssetRecord.csv]: eng_bulk_load-AssetRecord_NEW_2019-05-17 spreadsheet shows manuf sn as 00013; I believe that 0013 is correct in this csv, but should verify whether spreadsheet or this is correct.

Comment (rtravis-whoi) [inline bulk/eng_bulk_load-AssetRecord.csv]: eng_bulk_load-AssetRecord_NEW_2019-05-17 spreadsheet doesn't list a LEGACY_ASSET_UID for this one - may be a copy/paste error (same OL# listed for 3 entries in a row in the csv)

Comment (rtravis-whoi) [inline bulk/eng_bulk_load-AssetRecord.csv]: eng_bulk_load-AssetRecord_NEW_2019-05-17 spreadsheet has LEGACY_ASSET_UID of OL000583 - may be a copy/paste error (same OL# listed for 3 entries in a row in the csv)

Comment (rtravis-whoi) [inline bulk/eng_bulk_load-AssetRecord.csv]: Not sure if this one is supposed to have same ASSET_UID and LEGACY_ASSET_UID?

File: bulk/eng_bulk_load-AssetRecord.csv (hunk 8 above)
Comment (snwhite) [reply bulk/eng_bulk_load-AssetRecord.csv]: I don't want change OSU S/Ns, that's up to them.  I got a little overzealous in the Spreadsheet.

File: bulk/eng_bulk_load-AssetRecord.csv (hunk 9 above)
Comment (snwhite) [reply bulk/eng_bulk_load-AssetRecord.csv]: I don't want change OSU S/Ns, that's up to them. I got a little overzealous in the Spreadsheet.

File: bulk/eng_bulk_load-AssetRecord.csv (hunk 10 above)
Comment (snwhite) [reply bulk/eng_bulk_load-AssetRecord.csv]: I think this is correct.  They should be 4 digits.  I got a little overzealous in the Spreadsheet.

Comment (snwhite) [reply bulk/eng_bulk_load-AssetRecord.csv]: I think this is correct.  They should be 4 digits.  I got a little overzealous in the Spreadsheet.

Comment (snwhite) [reply bulk/eng_bulk_load-AssetRecord.csv]: That's how it was in GitHub.  I did not change that one.

File: bulk/eng_bulk_load-AssetRecord.csv [hunk 11]
```diff
@@ -60,14 +60,14 @@ CGCON-BDCL11-50009,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentr
 CGCON-BDCL11-60002,OL000594,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2016-0002,,,,,Woods Hole Oceanographic Institution
 CGCON-BDCL11-60701,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,60701,,,,DELET?,
```
Comment (pbrickley) [inline bulk/eng_bulk_load-AssetRecord.csv]: I don't know if this is correct.

File: bulk/eng_bulk_load-AssetRecord.csv [hunk 12]
```diff
@@ -60,14 +60,14 @@ CGCON-BDCL11-50009,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentr
...
 CGCON-BDCL11-70001,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2017-0001,,,,CP04OSSM-00006-DCL11,Woods Hole Oceanographic Institution
-CGCON-BDCL11-70009,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2017-0009,,,,,Woods Hole Oceanographic Institution
+CGCON-BDCL11-70009,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2017-009,,,,,Woods Hole Oceanographic Institution
```
Comment (pbrickley) [inline bulk/eng_bulk_load-AssetRecord.csv]: ok

File: bulk/eng_bulk_load-AssetRecord.csv [hunk 13]
```diff
@@ -171,7 +171,8 @@ CGCON-GLDRCP-00380,OL000355,Sensor,1,Mobile Asset Controller for Coastal Glider,
 CGCON-GLDRCP-00387,OL000356,Sensor,1,Mobile Asset Controller for Coastal Glider,GL387 Controller,Teledyne Webb,G2 Slocum Gliders,387,,,,,Woods Hole Oceanographic Institution
 CGCON-GLDRCP-00388,OL000357,Sensor,1,Mobile Asset Controller for Coastal Glider,GL388 Controller,Teledyne Webb,G2 Slocum Gliders,388,,,,,Woods Hole Oceanographic Institution
 CGCON-GLDRCP-00389,OL000358,Sensor,1,Mobile Asset Controller for Coastal Glider,GL389 Controller,Teledyne Webb,G2 Slocum Gliders,389,,,,,Woods Hole Oceanographic Institution
-CGCON-GLDRCP-00564,,Sensor,1,Mobile Asset Controller for Coastal Glider,GL564 Controller,Teledyne Webb,G2 Slocum Gliders,564,,,,Was Global; now Coastal,Woods Hole Oceanographic Institution
+CGCON-GLDRCP-00564,,Sensor,1,Mobile Asset Controller for Coastal Glider,GL564 Controller,Teledyne Webb,G2 Slocum Gliders,564,,,,,Woods Hole Oceanographic Institution
```
Comment (pbrickley) [inline bulk/eng_bulk_load-AssetRecord.csv]: Converted from Global unit to Coastal.  Was deployed as a global and as a coastal, so we need two entries.

File: bulk/eng_bulk_load-AssetRecord.csv [hunk 14]
```diff
@@ -171,7 +171,8 @@ CGCON-GLDRCP-00380,OL000355,Sensor,1,Mobile Asset Controller for Coastal Glider,
...
+CGCON-GLDRCP-00583,,Sensor,1,Mobile Asset Controller for Coastal Glider,GL564 Controller,Teledyne Webb,G2 Slocum Gliders,583,,,,Was Global; now Coastal,Woods Hole Oceanographic Institution
 CGCON-GLDRGP-00470,,Sensor,1,Mobile Asset Controller for Profiling Glider,,Teledyne Webb,G2 Slocum Gliders,470,,,,TO BE DELETED,Woods Hole Oceanographic Institution
```
Comment (pbrickley) [inline bulk/eng_bulk_load-AssetRecord.csv]: Why deleted?   Data was acquired from this unit before loss due to fouling at GA

File: bulk/eng_bulk_load-AssetRecord.csv [hunk 15]
```diff
@@ -171,7 +171,8 @@ CGCON-GLDRCP-00380,OL000355,Sensor,1,Mobile Asset Controller for Coastal Glider,
 CGCON-GLDRCP-00387,OL000356,Sensor,1,Mobile Asset Controller for Coastal Glider,GL387 Controller,Teledyne Webb,G2 Slocum Gliders,387,,,,,Woods Hole Oceanographic Institution
 CGCON-GLDRCP-00388,OL000357,Sensor,1,Mobile Asset Controller for Coastal Glider,GL388 Controller,Teledyne Webb,G2 Slocum Gliders,388,,,,,Woods Hole Oceanographic Institution
 CGCON-GLDRCP-00389,OL000358,Sensor,1,Mobile Asset Controller for Coastal Glider,GL389 Controller,Teledyne Webb,G2 Slocum Gliders,389,,,,,Woods Hole Oceanographic Institution
-CGCON-GLDRCP-00564,,Sensor,1,Mobile Asset Controller for Coastal Glider,GL564 Controller,Teledyne Webb,G2 Slocum Gliders,564,,,,Was Global; now Coastal,Woods Hole Oceanographic Institution
+CGCON-GLDRCP-00564,,Sensor,1,Mobile Asset Controller for Coastal Glider,GL564 Controller,Teledyne Webb,G2 Slocum Gliders,564,,,,,Woods Hole Oceanographic Institution
+CGCON-GLDRCP-00583,,Sensor,1,Mobile Asset Controller for Coastal Glider,GL564 Controller,Teledyne Webb,G2 Slocum Gliders,583,,,,Was Global; now Coastal,Woods Hole Oceanographic Institution
```
Comment (pbrickley) [inline bulk/eng_bulk_load-AssetRecord.csv]: OK.  
Converted from Global PG to Coastal PG but was never deployed as a global unit, only coastal.

File: bulk/eng_bulk_load-AssetRecord.csv [hunk 16]
```diff
@@ -171,7 +171,8 @@ CGCON-GLDRCP-00380,OL000355,Sensor,1,Mobile Asset Controller for Coastal Glider,
...
 CGCON-GLDRGP-00470,,Sensor,1,Mobile Asset Controller for Profiling Glider,,Teledyne Webb,G2 Slocum Gliders,470,,,,TO BE DELETED,Woods Hole Oceanographic Institution
 CGCON-GLDRGP-00514,OL000133,Sensor,1,Mobile Asset Controller for Profiling Glider,PG514 Controller,Teledyne Webb,G2 Slocum Gliders,514,,,,,Woods Hole Oceanographic Institution
```
Comment (pbrickley) [inline bulk/eng_bulk_load-AssetRecord.csv]: OK, but I can't verify old UID

File: bulk/eng_bulk_load-AssetRecord.csv [hunk 17]
```diff
@@ -171,7 +171,8 @@ CGCON-GLDRCP-00380,OL000355,Sensor,1,Mobile Asset Controller for Coastal Glider,
...
 CGCON-GLDRGP-00515,OL000134,Sensor,1,Mobile Asset Controller for Profiling Glider,PG515 Controller,Teledyne Webb,G2 Slocum Gliders,515,,,,,Woods Hole Oceanographic Institution
```
Comment (pbrickley) [inline bulk/eng_bulk_load-AssetRecord.csv]: Looks ok, but I can't verify old UID

File: bulk/eng_bulk_load-AssetRecord.csv [hunk 18]
```diff
@@ -381,7 +382,7 @@ CGCON-NDCL26-00042,,Sensor,0,Near Surface Instrument Frame Data Concentrator Log
 CGCON-NDCL26-00047,,Sensor,0,Near Surface Instrument Frame Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,47,,,,,
 CGCON-NDCL26-00048,,Sensor,0,Near Surface Instrument Frame Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,48,,,,,
 CGCON-NDCL26-00052,,Sensor,0,Near Surface Instrument Frame Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,52,,,,,
-CGCON-NDCL27-00001,OL000489,Sensor,0,Near Surface Instrument Frame Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,001,,,,CP01CNSM-00001-DCL27,Woods Hole Oceanographic Institution
+CGCON-NDCL27-00001,OL000489,Sensor,0,Near Surface Instrument Frame Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,0001,,,,CP01CNSM-00001-DCL27,Woods Hole Oceanographic Institution
```
Comment (pbrickley) [inline bulk/eng_bulk_load-AssetRecord.csv]: ok

File: bulk/eng_bulk_load-AssetRecord.csv [hunk 19]
```diff
@@ -412,8 +413,8 @@ CGCON-SIOCON-12368,OL000087 R00054,Sensor,0,Subsurface Mooring Platform Controll
 CGCON-SIOCON-12369,OL000022,Sensor,0,Subsurface Mooring Platform Controller,SIO Controller,SIO,Subsurface Mooring Controller,12369,,,,,Woods Hole Oceanographic Institution
 CGCON-SIOCON-12371,OL000102,Sensor,0,Subsurface Mooring Platform Controller,SIO Controller,SIO,Subsurface Mooring Controller,12371,,,,,Woods Hole Oceanographic Institution
 CGCON-SIOCON-12372,OL000021,Sensor,0,Subsurface Mooring Platform Controller,SIO Controller,SIO,Subsurface Mooring Controller,12372,,,,,Woods Hole Oceanographic Institution
-CGCON-SIOCON-12378,OL000089,Sensor,0,Subsurface Mooring Platform Controller,SIO Controller,SIO,Subsurface Mooring Controller,12378,,,,,Woods Hole Oceanographic Institution
-CGCON-SIOCON-13277,OL000089,Sensor,0,Subsurface Mooring Platform Controller,SIO Controller,SIO,Subsurface Mooring Controller,13277,,,,,Woods Hole Oceanographic Institution
+CGCON-SIOCON-12378,,Sensor,0,Subsurface Mooring Platform Controller,SIO Controller,SIO,Subsurface Mooring Controller,12378,,,,,Woods Hole Oceanographic Institution
```
Comment (pbrickley) [inline bulk/eng_bulk_load-AssetRecord.csv]: no idea

File: bulk/eng_bulk_load-AssetRecord.csv [hunk 20]
```diff
@@ -412,8 +413,8 @@ CGCON-SIOCON-12368,OL000087 R00054,Sensor,0,Subsurface Mooring Platform Controll
...
+CGCON-SIOCON-13277,OL000583,Sensor,0,Subsurface Mooring Platform Controller,SIO Controller,SIO,Subsurface Mooring Controller,13277,,,,,Woods Hole Oceanographic Institution
```
Comment (pbrickley) [inline bulk/eng_bulk_load-AssetRecord.csv]: no idea, really. but looks ok

**Comment by johnmlund** [comment]
JKUO and JLund reviewed the engineering bulk load spread sheet for the SIO SSM controller serial numbers and the SSM WFP serial numbers.  All appear to make sense!  Spot checking the Metadata list appears to match as well.

**Pull request merged**